packages/mds-attachment-service/package.json

```diff
@@ -14,7 +14,7 @@
   "scripts": {
     "build": "tsc --build tsconfig.build.json",
     "start": "pnpm watch server",
-    "pretest": "pnpm typeorm -- schema:drop",
+    "pretest": "env-cmd -f ../../.env pnpm typeorm -- schema:drop",
```

Comment:
I like how this looks. Perhaps out of scope for this PR, but I wonder if it would work in other scenarios as well, like:
```
"test": "env-cmd -f ../../.env jest --runInBand",
"ts-node": "env-cmd -f ../../.env node --experimental-repl-await -r ts-node/register",
```

Reply:
AFAIK jest already evaluates the env via DOTENV. For the ts-node scripts, it may make sense. Shall I include that here? My current objective is to make the tests run.